compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -1052,6 +1041,20 @@ linalg::LinalgLoopDistributionOptions getIREELinalgLoopDistributionOptions(
                                                                         dim),
           distributionMethod};
     }
+    if (splitDim) {
+      std::reverse(splitNumTiles.begin(), splitNumTiles.end());
+      auto delinearized = builder.create<affine::AffineDelinearizeIndexOp>(
+          loc, *splitDim, splitNumTiles, /*hasOuterBound=*/true);
+      for (auto [i, id, numTiles] :
+           llvm::enumerate(delinearized.getResults(), splitNumTiles)) {
+        // We iterate the delinearize results from slowest up to fastest, and
+        // we know that these are all the highest values of `dim. Tha tis,
+        // `i = 0` corresponds to `dim = numParallelDims - 1`.
```

Comment:
Typo. And maybe just spell out the `dimension`. Wrapping with backtick is more common for variables. However, there is no a variable named as `dim`.

```suggestion
        // we know that these are all the highest values of `dim. Tha tis,
        // `i = 0` corresponds to the `numParallelDims - 1`-th dimension.
```